loc/static/js/main.js

```diff
@@ -66,18 +66,17 @@ jQuery(function($){
         });
     });
 
-
-    $('#id_date_from').datepicker({defaultDate: new Date(start_year, 1-1, 1), showOn: "focus", yearRange: start_year+":"+end_year, currentText: '', changeMonth: true, changeYear: true, closeText: 'Done'});
-    $('#id_date_to').datepicker({defaultDate: new Date(end_year, 12-1, 31), showOn: "focus", yearRange: start_year+":"+end_year, currentText: '', changeMonth: true, changeYear: true});
+    $('#id_date_from').datepicker({defaultDate: new Date(selected_start_year, 1-1, 1), showOn: "focus", yearRange: min_year+":"+ max_year, currentText: '', changeMonth: true, changeYear: true, closeText: 'Done'});
```

Comment:
I'd like whitespace around the operators (e.g. `1-1` => `1 - 1`). Not sure whether it's worth a comment but even with ~20 years of working with JS I like noting that the `-1` is necessary because months are zero-based in the Date constructor.

Running prettier with the default config produces something like this:
```javascript
$("#id_date_from").datepicker({
    defaultDate: new Date(start_year, 1 - 1, 1),
    showOn: "focus",
    yearRange: start_year + ":" + end_year,
    currentText: "",
    changeMonth: true,
    changeYear: true,
    closeText: "Done"
});

```